lib/developer_portal/app/controllers/developer_portal/access_codes_controller.rb

```diff
@@ -30,7 +30,7 @@ def show
   private
 
   def cms_params
-    params.permit(:cms_token, :cms)
+    params.permit(:signature, :expires_at, :cms)
```

Comment:
Do we need this provided we will not update any model with these parameters? I thought permitting parmeters is only for stuff updating an active record object.

Reply:
I tried removing the `permit` but then the tests fail with this error:

```
ActionController::UnfilteredParameters: unable to convert unpermitted parameters to hash
```

Reply:
ahha, converting to hash also requires it